Release8.md

```diff
@@ -0,0 +1,185 @@
+# Release Notes for SCS Release 8
+
+The SCS Software (reference implementation) is released in version 8 on 2025-04-09.
+
+## Scope
+
+SCS continues to deliver a coordinated release cvoering the various modules that comprise
+the SCS reference implementation. SCS includes the latest proven upstream technologies and
+ensures that upgrades to R8 are smooth using the CI processes. The reference implementation
+with the default settings fulfills the requirements from SCS-compatibel IaaS v5.1 and
+KaaS v1.0 standards.
+
+## Component Versions and User-visible improvements (highlights)
+
+- Cluster API Provider OpenStack v0.12
+  - adds OpenStack Resource Controller (ORC) as external component
+- OpenStack 2024.2 is included in R8, but can be skipped in case Operators
+  want to make use of skipping over it and do directly from 2024.1 to 2025.1
```

Comment:
```suggestion
- OpenStack 2024.2 is included in R8, but can be skipped in case Operators
  want to make use of skipping over it and upgrade directly from 2024.1 to 2025.1
```